kiwi/builder/live.py

```diff
@@ -241,8 +241,8 @@ def create(self):
         # calculate size and decide if we need UDF
         if rootsize.accumulate_mbyte_file_sizes() > 4096:
             log.info('ISO exceeds 4G size, using UDF filesystem')
-            custom_iso_args['create_options'].append('-allow-limited-size')
-            custom_iso_args['create_options'].append('-udf')
+            custom_iso_args['create_options'].append('-iso-level')
+            custom_iso_args['create_options'].append('3')
```

Comment:
just for my own understanding; does iso-level 3 imply that the UDF filesystem is used ? This is important for the kiwi boot code in order to work properly